pkgs/by-name/sa/satysfi/package.nix

```diff
@@ -82,13 +86,16 @@ ocamlPackages.buildDunePackage rec {
       $out/share/satysfi/dist/fonts/Junicode.ttf
   '';
 
-  meta = with lib; {
+  meta = {
     homepage = "https://github.com/gfngfn/SATySFi";
     description = "Statically-typed, functional typesetting system";
     changelog = "https://github.com/gfngfn/SATySFi/blob/v${version}/CHANGELOG.md";
-    license = licenses.lgpl3Only;
-    maintainers = [ maintainers.mt-caret ];
-    platforms = platforms.all;
+    license = lib.licenses.lgpl3Only;
+    maintainers = with lib.maintainers; [
+      mt-caret
+      momeemt
```

Comment:
Could you add yourself in another commit?

Reply:
I split the other commit [`4e5f21d` (#372315)](https://github.com/NixOS/nixpkgs/pull/372315/commits/4e5f21d4eaadc824878563ff9d87d4a05974d8bf).